b2sdk/v2/transfer.py

```diff
@@ -7,29 +7,48 @@
 # License https://www.backblaze.com/using_b2_code.html
 #
 ######################################################################
+from concurrent.futures import Future, ThreadPoolExecutor
```

Comment:
should be a blank line above that

Reply:
My IDE automatically adds imports and `nox -s lint` doesn't check it... :/

Reply:
Fixed in [fc9de68](https://github.com/Backblaze/b2-sdk-python/pull/297/commits/fc9de68873550ac431cee0d257bb714d1ebee152)